fabric-item-api-v1/src/main/java/net/fabricmc/fabric/impl/item/ItemExtensions.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.item;
+
+import net.fabricmc.fabric.api.item.v1.EquipmentSlotProvider;
+
+public interface ItemExtensions {
+	EquipmentSlotProvider getEquipmentSlotProvider();
```

Comment:
prefix with `fabric_` since this is mixed into item

Reply:
Not necessary since it uses a Fabric API class in the signature.

Reply:
I added it anyway for consistency with other ducks.

Reply:
🦆